src/const.h

```diff
@@ -681,6 +681,46 @@ enum ReloadTypes_t : uint8_t
 	RELOAD_TYPE_WEAPONS,
 };
 
+enum MonsterIcon_t : uint8_t
+{
+	MONSTER_ICON_NONE,
+	MONSTER_ICON_VULNERABLE,
+	MONSTER_ICON_WEAKENED,
+	MONSTER_ICON_MELEE,
+	MONSTER_ICON_INFLUENCED,
+	MONSTER_ICON_FIENDISH,
+
+	MONSTER_ICON_LAST
+};
+
+enum CreatureIcon_t : uint8_t
+{
+	CREATURE_ICON_NONE,
+	CREATURE_ICON_CROSS_WHITE,
+	CREATURE_ICON_CROSS_WHITE_RED,
+	CREATURE_ICON_ORB_RED,
+	CREATURE_ICON_ORB_GREEN,
+	CREATURE_ICON_ORB_RED_GREEN,
+	CREATURE_ICON_GEM_GREEN,
+	CREATURE_ICON_GEM_YELLOW,
+	CREATURE_ICON_GEM_BLUE,
+	CREATURE_ICON_GEM_PURPLE,
+	CREATURE_ICON_GEM_RED,
+	CREATURE_ICON_PIGEON,
+	CREATURE_ICON_ENERGY,
+	CREATURE_ICON_POISON,
+	CREATURE_ICON_WATER,
+	CREATURE_ICON_FIRE,
+	CREATURE_ICON_ICE,
+	CREATURE_ICON_ARROW_UP,
+	CREATURE_ICON_ARROW_DOWN,
+	CREATURE_ICON_WARNING,
+	CREATURE_ICON_QUESTION,
+	CREATURE_ICON_CROSS_RED,
+
+	CREATURE_ICON_LAST
+};
+
```

Comment:
The `_LAST` values are not used and can be removed. `_NONE` also doesn't seem to make sense.

Reply:
_NONE ok

I thought about eliminating _LAST, but if it is used, it would also look ugly to use the last enum directly, better to use the name LAST